tests/h/activity/query_test.py

```diff
@@ -33,11 +33,9 @@ def test_returns_parse_results(self, parse, pyramid_request):
     def test_overrides_group_term_for_group_search_requests(
         self, parse, pyramid_request
     ):
-        """
-        If the query sent to a group search page includes a group, we override
-        it, because otherwise we'll display the union of the results for those
-        two groups, which makes no sense.
-        """
+        # If the query sent to a group search page includes a group, we override
+        # it, because otherwise we'll display the union of the results for those
+        # two groups, which makes no sense.
```

Comment:
Cheated in a few with this instead of adding a (IMO) unnecessary summary line for the tests docs

Reply:
I don't think we want to enforce docstrings in the tests at all. Comments are totally appropriate.